docs/sources/mimir/configure/configure-native-histograms-ingestion.md

```diff
@@ -7,20 +7,37 @@ weight: 160
 
 # Configure native histograms
 
-To enable support for ingesting Prometheus native histograms over the [remote write API]({{< relref "../references/http-api#remote-write" >}}) endpoint, set the flag `-ingester.native-histograms-ingestion-enabled=true` on ingesters.
+Prometheus native histograms ingestion is an **experimental** feature of Grafana Mimir.
 
-To enable support for querying native histograms together with [Grafana Mimir query sharding]({{< relref "../references/architecture/query-sharding" >}}), set the flag `-query-frontend.query-result-response-format=protobuf` on query frontends.
+You can configure native histograms ingestion via the Prometheus [remote write API]({{< relref "../references/http-api#remote-write" >}}) endpoint globally or per tenant.
 
-> **Note:** Native histograms is an experimental feature of Grafana Mimir.
+{{% admonition type="note" %}}
+To enable support for querying native histograms together with [Grafana Mimir query sharding]({{< relref "../references/architecture/query-sharding" >}}), make sure that the flag `-query-frontend.query-result-response-format` is set to its default `protobuf` value on query frontends.
+{{% /admonition %}}
 
-## Configure Prometheus to write native histograms to Grafana Mimir
+## Configure native histograms globally
 
-To enable experimental support for scraping and ingesting native histograms in Prometheus, [enable the feature](https://prometheus.io/docs/prometheus/latest/feature_flags/#native-histograms) with the flag `--enable-feature=native-histograms`.
+To enable ingesting Prometheus native histograms over the [remote write API]({{< relref "../references/http-api#remote-write" >}}) endpoint for all tenants, set the flag `-ingester.native-histograms-ingestion-enabled=true` on ingesters.
 
-To enable Prometheus remote write to send native histograms to Grafana Mimir, add the `send_native_histograms: true` parameter to your remote write configuration, for example:
+To limit the number of native histogram buckets per sample, set the `-validation.max-native-histogram-buckets` flag on distributors.
+The recommended value is 160 which is the default in the [OpenTelemetry SDK](https://opentelemetry.io/docs/specs/otel/metrics/sdk/) for exponential histograms, which are a similar concept in OpenTelemetry.
```

Comment:
```suggestion
The recommended value is `160`, which is the default in the [OpenTelemetry SDK](https://opentelemetry.io/docs/specs/otel/metrics/sdk/) for exponential histograms. The native histogram buckets are similar in concept to exponential histograms in OpenTelemetry.
```
Spell out that last sentence. Please vet this for technical accuracy, because my change is a guess and has high potential for being inaccurate in this case.